makelib/k8s_tools.mk

```diff
@@ -148,20 +139,6 @@ $(KUSTOMIZE):
 	@rm -fr $(TOOLS_HOST_DIR)/tmp-kustomize
 	@$(OK) installing kustomize $(KUSTOMIZE_VERSION)
 
-# olm-bundle download and install
-$(OLMBUNDLE):
-	@$(INFO) installing olm-bundle $(OLMBUNDLE_VERSION)
-	@curl -fsSLo $(OLMBUNDLE) https://github.com/upbound/olm-bundle/releases/download/$(OLMBUNDLE_VERSION)/olm-bundle_$(SAFEHOSTPLATFORM) || $(FAIL)
-	@chmod +x $(OLMBUNDLE)
-	@$(OK) installing olm-bundle $(OLMBUNDLE_VERSION)
-
-# up download and install
-$(UP):
```

Comment:
I don't think it's safe to remove the `up` tool from this build submodule yet. The family providers still use `up xpkg batch` to publish batches of family providers more efficiently. e.g.:

* provider-upjet-aws uses this crossplane/build submodule: https://github.com/crossplane-contrib/provider-upjet-aws/blob/main/.gitmodules#L7
* it depends on `up` to run `up xpkg batch`: https://github.com/crossplane-contrib/provider-upjet-aws/blob/main/cluster/images/provider-aws/Makefile#L60 

I think we have to leave `up` in this submodule until that is resolved 🤔